rxbindings/src/test/java/com/amplifyframework/rx/RxStorageBindingTest.java

```diff
@@ -120,7 +120,7 @@ public void downloadFileReturnsResult() throws InterruptedException {
             Consumer<StorageTransferProgress> progressConsumer = invocation.getArgument(indexOfProgressConsumer);
             Consumer<StorageDownloadFileResult> resultConsumer = invocation.getArgument(indexOfResultConsumer);
 
-            Observable.interval(100, TimeUnit.MILLISECONDS)
+            Observable.interval(100, 100, TimeUnit.MILLISECONDS)
```

Comment:
The three-argument version of this method accepts a `long initialDelay` as the first parameter. ([Refer to RxJava3 `Observable.interval(long, long TimeUnit)` documentation](http://reactivex.io/RxJava/3.x/javadoc/io/reactivex/rxjava3/core/Observable.html#interval-long-long-java.util.concurrent.TimeUnit-).)